src/cascadia/TerminalApp/App.cpp

```diff
@@ -238,10 +248,6 @@ namespace winrt::TerminalApp::implementation
 
         const auto buttonText = _resourceLoader.GetLocalizedString(L"Ok");
 
-        gettingStartedLink.Foreground(blueBrush);

```

Comment:
Are these the only uses of blueBrush? Should you comment out the brush creation too since it's unused right now?

Also, why is it OK to make these not blue anymore?

Reply:
@cinnamon-msft could probably explain why they were manually made blue before. I was just trying things and found the hyperlink was blue by default, so I didn't think we needed to manually set the color. I can clean up the brush though.